src/components/header/JourneySection.tsx

```diff
@@ -29,11 +29,11 @@ const JourneySection = () => {
 
           <div className="m-2 p-3 bg-gray-400 text-white flex rounded-lg justify-around text-xl  items-center">
```

Comment:
Can you increase the border radius to match the design

Reply:
fixed